xcode/Subconscious/Shared/Components/Common/StoryOnThisDayView.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  StoryOnThisDayView.swift
+//  Subconscious (iOS)
+//
+//  Created by Ben Follington on 30/9/2022.
+//
+
+import Foundation
+import SwiftUI
+
+/// A story is a single update within the FeedView
+struct StoryOnThisDayView: View {
+    var story: StoryOnThisDay
+    var action: (EntryLink, String) -> Void
+    
+    func readableDescription() -> String {
+        switch story.timespan {
+        case .aYearAgo:
+            return "On this day a year ago"
+        case .sixMonthsAgo:
+            return "6 months ago"
+        case .aMonthAgo:
+            return "One month ago"
+        case .inTheLastWeek:
+            return "Modified in the last week"
+        case .inTheLastDay:
+            return "Modified in the last day"
+        }
+    }
```

Comment:
I had a go at phrasing these in a feed-friendly way but they seem a bit clunky still.